### PR TITLE
feat: hidden feature -- earlier sprints

### PR DESCRIPTION
**Pull request by mcdurdin**
This allows you to add a `?sprint=P8S4` parameter to view sprint data for P8S4. The only information that is really relevant for earlier sprints is the contributions data. It's not all that beautiful, but I think it all works.

### CODE REVIEW

**Comment by darcywong00** [comment]
> This allows you to add a `?sprint=P8S4` parameter to view sprint data for P8S4

Can you add this to the readme.md?

File: current-sprint.js [hunk 1]
```diff
@@ -1,22 +1,32 @@
...
-    if(m.node.dueOn == null) return a;
-    if(a == null || a.node.dueOn == null) return m;
-    if(new Date(a.node.dueOn) < new Date(m.node.dueOn)) return a;
-    return m;
-  });
-  if(phase == null) {
-    return null;
+  let phase = null;
+
+  if(ghdata.milestoneDueOn) {
```
Comment (darcywong00) [inline current-sprint.js]: When I ran node code.js, I got this
```
 UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'milestoneDueOn' of undefined
    at Object.exports.getCurrentSprint (D:\src\sites\status.keyman.com\current-sprint.js:6:13)
```

Comment (darcywong00) [reply current-sprint.js]: ah, likely cause I haven't set up tokens on this machine

Comment (mcdurdin) [reply current-sprint.js]: Yeah, some of the error case pathways are not complete for this site; don't really want to expend too much effort on them for now.

**Comment by mcdurdin** [comment]
> 2\. I guess this doesn't combine with `?c=1`. Looks like only the first paramter got used between:

It does work :) it's only useful with c=1. However, it loads the default sprint first before reloading with the requested sprint (not ideal, but workable for now).

**Pull request merged**